bindata/v3.11.0/openshift-apiserver/ds.yaml

```diff
@@ -36,9 +37,21 @@ spec:
         terminationMessagePolicy: FallbackToLogsOnError
         image: ${IMAGE}
         imagePullPolicy: IfNotPresent
-        command: ["openshift-apiserver", "start"]
+        command: ["/bin/bash", "-ec"]
         args:
-        - "--config=/var/run/configmaps/config/config.yaml"
+          - |
+            echo $$ > /var/run/watchdog/pid
```

Comment:
You probably noticed, these dollar signs seem to be escaped, so they won’t be interpreted.

I wonder if we could either use realpath of /proc/self or get the value of either /proc/self/stat{,us}. Starts to be a bit hackish.

Reply:
they should be escaped and passed to bash, which then interprets them.

Reply:
yaml doesn't escape I hope

Reply:
but I agree that `W0827 11:29:32.124287      29 cmd.go:211] Unable to parse pid file /var/run/watchdog/pid: strconv.Atoi: parsing "$": invalid syntax` is convincing.

Reply:
I’m just assuming given the watchdog reports “/var/run/watchdog/pid: strconv.Atoi: parsing "$": invalid syntax”

Reply:
`$$$$` does the trick. Is that yaml escaping of dollar?

Reply:
Hard to tell, I did not think dollar sign has a special meaning for yaml files :/

Reply:
try `kubectl run --restart=Never -i -t busybox --image busybox -- /bin/sh -ec 'echo $$$$'`